conan/tools/microsoft/msbuild.py

```diff
@@ -1,17 +1,26 @@
+from conans.client.tools.oss import cpu_count
 from conans.errors import ConanException
 
 
 def msbuild_verbosity_cmd_line_arg(conanfile):
-    verbosity = conanfile.conf["tools.microsoft"].msbuild_verbosity
+    verbosity = conanfile.conf["tools.microsoft.msbuild"].verbosity
```

Comment:
@memsharded ,  Is `conf["tools.microsoft"].msbuild_verbosity` intended? I see others like `conf["tools.microsoft.msbuild"].vs_version` and `conf["tools.microsoft.msbuildtoolchain"].compile_options` and I feel like `conf["tools.microsoft.msbuild"].verbosity` is more aligned with them.

Reply:
👍   Moved to its own PR: https://github.com/conan-io/conan/pull/8692